src/pack.rs

```diff
@@ -195,13 +195,17 @@ pub fn pack_instance(raw_module: Vec<u8>, mut ctor_module: elements::Module, tar
             ])).build()
             .build()
         .build();
+    
+    if let TargetRuntime::Substrate(_) = target {
+        return Ok(new_module)
+    }
```

Comment:
I'm not sure we want to return here. In substrate after migration to the new model we don't need to use packer anymore. So I'd rather skip `pack_instance` for substrate contracts altogether.